example/java/build_library.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 cd "$(dirname "$0")"
 
+echo "Usage: $0 [JAVA_PACKAGE]"
+echo "Example: $0"
+echo "Example: $0 org.iroha"
+echo ""
+if [[ $# == 1 ]]; then
+    PACKAGE=$1
+    echo PACKAGE=${PACKAGE}
+	
+    #replace line
+    #    find_package(JNI REQUIRED)
```

Comment:
I haven't tested it yet, but it doesn't seem to be a working solution. JNI package still should be found, I guess